lib/irb.rb

```diff
@@ -775,16 +775,25 @@ def signal_status(status)
       end
     end
 
+    def truncate_prompt_main(str, size = 32, omission: '...') # :nodoc:
```

Comment:
Do we have plan to specify `size` and `omission` in follow up PRs? If not, maybe we can specify them in constants first? (omission can use `.freeze`)

Reply:
>  follow up PRs?

no plan, I'll use constants :+1: